packages/web/components/MobileFooter.tsx

```diff
@@ -0,0 +1,175 @@
+import { Button, Flex, Image, Stack } from '@metafam/ds';
+import MetaBoxButton from 'assets/drawer/box.button.bg.png';
+import MetaForum from 'assets/drawer/forum.png';
+import MetaPlayers from 'assets/drawer/players.png';
+import MetaQuests from 'assets/drawer/quests.png';
+import MetaRaids from 'assets/drawer/raids.png';
+import MetaGameLogo from 'assets/logo.png';
+import { MetaLink } from 'components/Link';
+import React from 'react';
+
+import { DrawerItems } from './DrawerItems';
+
+const MenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
+  children,
+  href,
+  isExternal,
+}) => {
+  return (
+    <MetaLink zIndex="2" href={href} isExternal={isExternal}>
+      <Button
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        width="20vw"
+        height="5rem"
+        textDecoration="none"
+        variant="link"
+        p="1"
+        fontFamily="mono"
+        color="whiteAlpha.700"
+      >
+        {children}
+      </Button>
+    </MetaLink>
+  );
+};
+
+const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
```

Comment:
These components are being duplicated in MobileFooter and in PageHeader. Should refactor to make these generic and re-usable in separate files instead of being in the same file as the Footer (rule of thumb: only 1 component per file).

Then, consolidate the implementation of the AppDrawer to a single component instead of having the same code duplicated.